Expressions.Rmd

```diff
@@ -380,6 +380,10 @@ The following table summarises the appearance of the different expression subtyp
 
 Both base R and rlang provide functions for testing for each type of input, although the types covered are slightly different. You can easily tell them apart because all the base functions start with `is.` and the rlang functions start with `is_`.
 
+\newpage
+
+<!-- New page so that there's no beak inside the table -->
```

Comment:
In this case it would be better to float the table (and then reference it in the text)

Reply:
I tried, but the problem is that pandoc converts markdown tables to latex using the "longtable" environment, which means that allows tables that span multiple pages and in this case it 
![image](https://user-images.githubusercontent.com/8617595/54938392-0a6f2e00-4f05-11e9-9602-6bff0a654bfb.png)

From what I've learned, this is backed into pandoc so there's no easy way to change it.

Reply:
Ah got it, thanks for the investigation